client/rest-high-level/src/main/java/org/elasticsearch/client/indices/DataStreamsStatsRequest.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.client.indices;
+
+import org.elasticsearch.action.support.IndicesOptions;
+import org.elasticsearch.client.Validatable;
+
+public class DataStreamsStatsRequest implements Validatable {
```

Comment:
Should this be on the HLRC request class or the transport request? I'm assuming transport, but wanted to double check.

Reply:
Actually, it looks like this interface is extended already on `BroadcastRequest` which is the super class on the transport request. Is this something that still needs to be done on the client side?

Reply:
> Actually, it looks like this interface is extended already on `BroadcastRequest` which is the super class on the transport request. Is this something that still needs to be done on the client side?

Ah, you're right. I missed that. Because the action name begins with `indices:monitor`, it should automatically be picked up by the `monitor` and `manage` index privileges which I think is appropriate. Maybe just a REST test to verify that, then.